model/collection.py

```diff
@@ -745,7 +745,12 @@ def restrict_to_ready_deliverable_works(
 
         # Only find books with available licenses or books from self-hosted collections using MirrorUploader
         query = query.filter(
-            or_(LicensePool.licenses_owned > 0, LicensePool.open_access, LicensePool.self_hosted)
+            or_(
+                LicensePool.licenses_owned > 0,
+                LicensePool.open_access,
+                LicensePool.unlimited_access,
```

Comment:
I'm amazed that this works, since `unlimited_access` is not a database field, but I checked and it does work.

Reply:
It works because of `@hybrid_property` decorator, it makes it act like a real database field